### PR TITLE
List Pkl GitHub repositories in README

### PR DESCRIPTION
**Pull request by translatenix**
Motivation: Discoverability, convenience

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -41,3 +41,73 @@ We'd love to hear from you!
...
+
+|https://github.com/apple/pkl.tmbundle[`apple/pkl.tmbundle`]
+|
+
+|https://github.com/apple/rules_pkl[`apple/rules_pkl`]
+|
+
+|https://github.com/apple/tree-sitter-pkl[`apple/tree-sitter-pkl`]
+|Tree-sitter parser for Pkl
+|===
```
Comment (bioball) [inline README.adoc:113]: How about just linking to https://github.com/orgs/apple/repositories?&q=pkl&type=all? That way, we don't need to worry about syncing our growing list of repos with this list.

Comment (translatenix) [reply README.adoc:113]: That page is less terse and a link away, but it’s definitely better than nothing.

The inspiration for this PR came from here:
* https://github.com/graalvm#graalvms-github-repositories
* https://github.com/oracle/graal#related-repositories

File: README.adoc [hunk 2]
```diff
@@ -41,3 +41,73 @@ We'd love to hear from you!
...
+|Examples for using Pkl within Swift applications
+
+|https://github.com/apple/pkl-vscode[`apple/pkl-vscode`]
+|Pkl language support for VS Code
+
+|https://github.com/apple/pkl.tmbundle[`apple/pkl.tmbundle`]
+|
+
+|https://github.com/apple/rules_pkl[`apple/rules_pkl`]
+|
```
Comment (bioball) [inline README.adoc:109]:
```suggestion
| Bazel build rules for Pkl
```

File: README.adoc [hunk 3]
```diff
@@ -41,3 +41,73 @@ We'd love to hear from you!
...
+
+[%autowidth]
+|===
+|Name |Description
+
+|`apple/pkl`
+|A configuration as code language with rich validation and tooling.
+
+|https://github.com/apple/pkl-evolution[`apple/pkl-evolution`]
+|
```
Comment (bioball) [inline README.adoc:55]:
```suggestion
|Suggested Pkl Improvements, Changes, or Enhancements (SPICEs)
```

**Review by bioball — APPROVED** [review]
LGTM

File: README.adoc (hunk 1 above)
Comment (bioball) [reply README.adoc:113]: Okay; works with me!

**Pull request merged**